sentry_sdk/_types.py

```diff
@@ -220,7 +220,9 @@ class SDKInfo(TypedDict):
         tuple[None, None, None],
     ]
 
+    # TODO: Make a proper type definition for this (PRs welcome!)
```

Comment:
[nit] I'd suggest we just remove these `TODO` comments and make GitHub issues for them; I think `TODO` comments will just get ignored 

```suggestion
```

Reply:
I will keep them. Maybe someone wants to earn some internet points and creates nice types here :-)

Reply:
okay fair enough 😂